spec/apruve/resources/order_spec.rb

```diff
@@ -141,6 +141,33 @@
     end
   end
 
+  describe '#find_all' do
+    context 'successful response' do
+      let! (:stubs) do
+        faraday_stubs do |stub|
+          stub.get('/api/v4/orders') { [200, {}, '{}'] }
```

Comment:
Small thing, but the successful response for orders here would return a JSON array, instead of a single JSON object - the second argument in the stubbed response would be `'[]'`.